.github/workflows/pull_request.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     - run: gem install bundler
     - run: bundle install
     - run: bundle exec rubocop
-    - run: bundle exec rspec
+    - run: rake test
```

Comment:
maintainer: please don't call `bundle exec rspec`. It fails even in your local after merging this change.